demisto_sdk/scripts/init_validation_script.py

```diff
@@ -0,0 +1,540 @@
+        "content_type": "IncidentType",
+    },
+    "14": {
+        "import": "from demisto_sdk.commands.content_graph.objects.indicator_field import IndicatorField",
+        "content_type": "IndicatorField",
+    },
+    "15": {
+        "import": "from demisto_sdk.commands.content_graph.objects.indicator_type import IndicatorType",
+        "content_type": "IndicatorType",
+    },
+    "16": {
+        "import": "from demisto_sdk.commands.content_graph.objects.layout_rule import LayoutRule",
+        "content_type": "LayoutRule",
+    },
+    "17": {
+        "import": "from demisto_sdk.commands.content_graph.objects.layout import Layout",
+        "content_type": "Layout",
+    },
+    "18": {
+        "import": "from demisto_sdk.commands.content_graph.objects.modeling_rule import ModelingRule",
+        "content_type": "ModelingRule",
+    },
+    "19": {
+        "import": "from demisto_sdk.commands.content_graph.objects.parsing_Rule import ParsingRule",
+        "content_type": "ParsingRule",
+    },
+    "20": {
+        "import": "from demisto_sdk.commands.content_graph.objects.report import Report",
+        "content_type": "Report",
+    },
+    "21": {
+        "import": "from demisto_sdk.commands.content_graph.objects.test_playbook import TestPlaybook",
+        "content_type": "TestPlaybook",
+    },
+    "22": {
+        "import": "from demisto_sdk.commands.content_graph.objects.trigger import Trigger",
+        "content_type": "Trigger",
+    },
+    "23": {
+        "import": "from demisto_sdk.commands.content_graph.objects.widget import Widget",
+        "content_type": "Widget",
+    },
+    "24": {
+        "import": "from demisto_sdk.commands.content_graph.objects.generic_definition import GenericDefinition",
+        "content_type": "GenericDefinition",
+    },
+    "25": {
+        "import": "from demisto_sdk.commands.content_graph.objects.generic_field import GenericField",
+        "content_type": "GenericField",
+    },
+    "26": {
+        "import": "from demisto_sdk.commands.content_graph.objects.generic_module import GenericModule",
+        "content_type": "GenericModule",
+    },
+    "27": {
+        "import": "from demisto_sdk.commands.content_graph.objects.generic_type import GenericType",
+        "content_type": "GenericType",
+    },
+    "28": {
+        "import": "from demisto_sdk.commands.content_graph.objects.xsiam_dashboard import XSIAMDashboard",
+        "content_type": "XSIAMDashboard",
+    },
+    "29": {
+        "import": "from demisto_sdk.commands.content_graph.objects.xsiam_report import XSIAMReport",
+        "content_type": "XSIAMReport",
+    },
+}
+
+VALIDATION_TEMPLATE = """
+$imports
+
+$supported_content_types
+
+
+$class_declaration
+    error_code = "$error_code"
+    description = "$error_description"
+    error_message = "$error_message"
+    fix_message = "$fix_message"
+    related_field = "$related_field"
+    is_auto_fixable = $is_auto_fixable
+    $expected_git_statuses$graph
```

Comment:
is this right?

Reply:
Yes, since these two are not mandatory, `expected_git_statuses` will end with a new line mark if added..